ArduPlane/systemid.cpp

```diff
@@ -0,0 +1,243 @@
+#include "systemid.h"
+
+#if AP_PLANE_SYSTEMID_ENABLED
+
+#include <AP_Math/control.h>
+#include "Plane.h"
+
+/*
+  handle systemid via an auxillary switch
+ */
+
+const AP_Param::GroupInfo AP_SystemID::var_info[] = {
+
+    // @Param: _AXIS
+    // @DisplayName: System identification axis
+    // @Description: Controls which axis are being excited.  Set to non-zero to see more parameters
+    // @User: Standard
+    // @Values: 0:None, 1:Input Roll Angle, 2:Input Pitch Angle, 3:Input Yaw Angle, 4:Recovery Roll Angle, 5:Recovery Pitch Angle, 6:Recovery Yaw Angle, 7:Rate Roll, 8:Rate Pitch, 9:Rate Yaw, 10:Mixer Roll, 11:Mixer Pitch, 12:Mixer Yaw, 13:Mixer Thrust, 14:Measured Lateral Position, 15:Measured Longitudinal Position, 16:Measured Lateral Velocity, 17:Measured Longitudinal Velocity, 18:Input Lateral Velocity, 19:Input Longitudinal Velocity
```

Comment:
Should add "VTOL" to these, would be good to support in FW flight in the future.

Reply:
I was planning to re-use the same axes (where relevant, can't do lateral vel and pos).

Reply:
remove values 14-19.  I did not implement them in my version of sysid.  it doesn't look like you did either.  Lets keep it to the attitude controller.  We can expand to the position controller later.

Reply:
done